confluent_kafka/src/Consumer.c

```diff
@@ -743,6 +743,75 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 }
 
 
+static PyObject *Consumer_consume (Handle *self, PyObject *args,
+                                        PyObject *kwargs) {
+        size_t num_messages = 1;
+        double tmout = -1.0f;
+        static char *kws[] = { "num_messages", "timeout", NULL };
+        rd_kafka_message_t **rkmessages;
+        PyObject *msglist;
+        rd_kafka_queue_t *rkqu = NULL;
+        CallState cs;
+
+        if (!self->rk) {
+                PyErr_SetString(PyExc_RuntimeError,
+                                "Consumer closed");
+                return NULL;
+        }
+
+        if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|Id", kws,
+					 (unsigned int *)&num_messages, &tmout))
```

Comment:
It is not safe to cast to integer pointers of possibly other size.
Use `unsigned int` for num_message's type directly to avoid the case.